### PR TITLE
Fix dependency license's NOTICE and binary jar included issues in the source release.

### LINKED ISSUE #9512: [Bug] Fix dependency license's NOTICE and binary jar included issues in the source release.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

We received reports from incubator discussion

```
So taking a quick look at a SkyWalking release [1] I can see a couple of minor issues with LICNSE and NOTICE. You may want to fix these.

Hints:
Is OpenCensus mentioned in LICENSE?
Does prometheus client_model have a NOTICE file?
Does mavin-wrapper have a NOTICE file?

Also there is a compiled code (a jar) in the source release. Source release cannot include compiled code.

Kind Regards,
Justin

1. https://www.apache.org/dyn/closer.cgi/skywalking/9.1.0/apache-skywalking-apm-9.1.0-src.tgz <https://www.apache.org/dyn/closer.cgi/skywalking/9.1.0/apache-skywalking-apm-9.1.0-src.tgz>

```

### What you expected to happen

We should remove the jar file from the source tar.

### How to reproduce

Just repackage the v9.1.0 tag for a source release.

### Anything else

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by CalvinKirs**
I checked: https://github.com/apache/skywalking/blob/77ba2d553d77a4acacd8b2498cb09ea983a9baf4/LICENSE#L215-L225
Only `maven-wrapper` and `Prometheus` need to add notice

this closed #9512

### CODE REVIEW

**Comment by wu-sheng** [comment]
I think OpenCensus was being asked too. Could you recheck that?

**Comment by CalvinKirs** [comment]
> I think OpenCensus was being asked too. Could you recheck that?

Hi, I checked, PTAL, thx

**Comment by CalvinKirs** [comment]
But about this, `oap-server/server-receiver-plugin/receiver-proto/src/main/proto/satellite/envoy/accesslog/v3/als.proto` I can't be sure, could you help me take a look?

**Comment by wu-sheng** [comment]
als.proto is from envoy project.

**Comment by CalvinKirs** [comment]
> als.proto is from envoy project.

Thanks, done

File: LICENSE [hunk 1]
```diff
@@ -217,6 +217,8 @@ The text of each license is the standard Apache 2.0 license.
 
    proto files from cncf/udpa: https://github.com/cncf/udpa Apache 2.0
    proto files from envoyproxy/data-plane-api: https://github.com/envoyproxy/data-plane-api  Apache 2.0
+   proto files from satellite/envoy/: https://github.com/envoyproxy/envoy  Apache 2.0
```
Comment (wu-sheng) [inline LICENSE:220]: I think this is skywalking's als fork? @mrproliu 
If so, this is not from envoy. And I can see we have envoy data panel proto licences already.

Comment (wu-sheng) [reply LICENSE:220]: I checked, this is a shell of envoy ALS, and it belongs to SkyWalking.

Comment (wu-sheng) [reply LICENSE:220]:
```suggestion
```

**Pull request merged**